internal/providers/terraform/aws/vpn_connection.go

```diff
@@ -0,0 +1,56 @@
+package aws
+
+import (
+	"github.com/infracost/infracost/internal/schema"
+
+	"github.com/shopspring/decimal"
+)
+
+func GetVPNConnectionRegistryItem() *schema.RegistryItem {
+	return &schema.RegistryItem{
+		Name:  "aws_vpn_connection",
```

Comment:
@robh007 https://aws.amazon.com/transit-gateway/pricing/ shows `Price per GB of data processed` too, should that be included?